Sources/StreamVideoSwiftUI/ViewFactory.swift

```diff
@@ -149,27 +149,44 @@ public protocol ViewFactory: AnyObject {
         callSettings: Binding<CallSettings>,
         call: Call?
     ) -> LocalParticipantViewModifierType
+
+    associatedtype BottomParticipantsBarLayoutComponentType: View
```

Comment:
I'm not convinced that we should expose these as customization slots. If you want to implement a different form of a scrollable list, then it will impact other parts of the view as well, and in that case, building the UI from scratch using our building blocks should be the recommended way.

Reply:
My train of thought here was that if someone wants to build a custom layout (with the horizontal bar on top instead at the botom, like zoom) they don't have to build everything from scratch.

Reply:
Yes, that's why we will have the 2 new components as public - customers can re-use them as building blocks. 
For the slots, it's a bit tricky, since as soon as we expose it, we can't remove it.

Reply:
I got it 👍